testdata/default/repros/Issue10302.t.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.18;
+
+import "ds-test/test.sol";
+import "cheats/Vm.sol";
+
+contract A {
+    function foo() public pure returns (bool) {
+        return true;
+    }
+}
+
+contract Issue10302Test is DSTest {
+    Vm constant vm = Vm(HEVM_ADDRESS);
+
+    function testDelegateFails() external {
+        vm.createSelectFork("sepolia");
```

Comment:
This might fail if this address ever have a zero balance. Maybe best to specify a fork number.